app/src/App.js

```diff
@@ -13,30 +14,16 @@ import ScrabbleHand from './components/ScrabbleHand';
 import ScrabbleBoard from './components/ScrabbleBoard';
 
 function App() {
-  const [currentBoard, setCurrentBoard] = useState(Array(15).fill(Array(15).fill(null)));
 
-  useEffect(() => {
-    const fakeTile = { value: 'A', score: 1 };
-    setCurrentBoard(currentBoard => {
-      const newBoard = currentBoard.map((row, rowIndex) => {
-        return row.map((element, columnIndex) => {
-          if (rowIndex === 3 && columnIndex === 2) {
-            return fakeTile;
-          }
-          return element;
-        });
-      }
-      );
-      return newBoard;
-    });
-  }, []);
+  const { currentBoard, currentTile } = useContext(GameContext);
 
   return (
     <BrowserRouter>
       <div className="page-container">
         <div className="content-wrap">
           <Header />
           <ScrabbleTile value={'B'} score={2} />
+          {currentTile && `The current tile value is ${currentTile.value} and score is ${currentTile.score}`}
```

Comment:
Ez csak ideiglenesen van itt? Én úgy képzeltem, hogy a ScrabbleHand komponensben a különleges border miatt egyértelmű lesz majd, hogy melyik a currentTile. De ide is rakhatsz valamit, csak akkor inkább magát a komponenst, ne egy ilyen szöveget (de ez csak stilisztikai hozzászólás a részemről)

Reply:
Ja bocs, ezt csak kiprobaltam, hogy megjeleniti-e, de persze, valami borderrel majd jelzem hogy melyik van kijelolve. De szerintem nem eleg a Hand komponensben, mert lehet, hogy majd a boardon is ujra ki kell jelolnom, ha megsem akarom lerakni nem? Nem kene inkabb a Tile komponensben siman?